ext/hal/nxp/mcux/middleware/wireless/framework_5.3.3/OSAbstraction/Interface/fsl_os_abstraction.h

```diff
@@ -140,6 +141,13 @@ typedef enum osaStatus_tag
 #define OSA_TASK_PRIORITY_MIN (15)  
 #define SIZE_IN_UINT32_UNITS(size) (((size) + sizeof(uint32_t) - 1) / sizeof(uint32_t)) 
 
+/*!
+ * @brief To provide unified task piority for upper layer, OSA layer makes conversion.
+ */
+#define configMAX_PRIORITIES            (CONFIG_NUM_COOP_PRIORITIES)
```

Comment:
We don't typically change files in `ext/` directly, instead importing a new revision from upstream.

Reply:
Understood. Part of the problem is the initial version of this file only included the minimum that the XCVR driver needed (before my time) so I continued adding the functions needed by the BLE link layer library. My preference right now is to not include all of the abstraction as we will likely not be pulling in any additional code from NXP that will need it.